awx/conf/views.py

```diff
@@ -72,7 +72,7 @@ class SettingSingletonDetail(RetrieveUpdateDestroyAPIView):
 
     def get_queryset(self):
         self.category_slug = self.kwargs.get('category_slug', 'all')
-        all_category_slugs = settings_registry.get_registered_categories(features_enabled=get_licensed_features()).keys()
+        all_category_slugs = list(settings_registry.get_registered_categories(features_enabled=get_licensed_features()).keys())
```

Comment:
I think we can drop the cast to `list`

PS: adding this here for a different pattern other them casting the return value of `items`

Reply:
Later, we modify this data structure as a list:
```
>>> x = {}.keys()
>>> x.remove
Traceback (most recent call last):
  File "<console>", line 1, in <module>
AttributeError: 'dict_keys' object has no attribute 'remove'
```